tests/ethereum/main.rs

```diff
@@ -143,28 +142,13 @@ fn run_test_unit(path: &Path, unit: &TestUnit) {
                 // EIP-2681
                 (Some("TR_NonceHasMaxValue"), Ok(exec_results)) => {
                     assert!(exec_results.len() == 1);
-                    assert!(match exec_results[0].result.clone() {
-                        ExecutionResult::Success {
-                            output: Output::Create(b, None),
-                            ..
-                        } => b == Bytes::new(),
-                        _ => false,
-                    });
+                    let (receipt, _state) = &exec_results[0];
+                    assert!(receipt.status);
                 }
                 // Special cases where REVM returns `Ok` instead of `Err` on unsupported features.
```

Comment:
We're already skipping it in the error case, let's just bite the bullet for now:
```
// Skipping special cases where REVM returns `Ok` instead on unsupported features.
(Some("TR_TypeNotSupported"), Ok(_)) => {}
```

Reply:
updated the comment